<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шахматы клиент-сервер</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ButClientRun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключиться к серверу</string>
    </property>
   </widget>
   <widget class="QLabel" name="labStepInfo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Сервер</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edHost">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetChat" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>281</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="textChat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>481</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButSend">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>500</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cbUseBot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим бота</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
